client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -1,268 +1,94 @@
 @import 'assets/stylesheets/gutenberg-base-styles';
 @import '~@wordpress/base-styles/colors';
-
-$design-selector-grid-gap: 4.5em;
-
-$design-selector-max-width: 1100px;
-$design-selector-selection-space: 285px + $header-height;
-$page-selector-width-offset: 20px;
-$page-selector-shadow-size: 12px;
-
-.design-selector__title {
-	color: $dark-gray-800;
-	font-size: 2em;
-	font-weight: bold;
-	margin-bottom: 0.25em;
-	text-align: center;
-}
-
-.design-selector__subtitle {
-	color: $dark-gray-800;
-	font-size: 1.3em;
-	text-align: center;
-}
-
-.design-selector__grid-container {
-	position: absolute;
-	padding-bottom: 48px;
-	padding-top: 19px;
-
-	&.is-page-selector-open {
-		overflow-y: hidden;
-		height: 100vh;
-	}
-}
-
-.design-selector__grid {
-	display: grid;
-	grid-gap: $design-selector-grid-gap;
-
-	@include breakpoint( '>660px' ) {
-		grid-template-columns: 1fr 1fr;
-	}
-}
-
-.design-selector__page-layout-backdrop {
+@import '../../mixins';
+
+@font-face {
```

Comment:
I think this is already defined in another scss file lower in the tree and can be removed here.